docs/content/direct/combined-status.md

```diff
@@ -8,38 +8,85 @@ It may make sense to expand the latter into a general feature in the future, acc
+
+1. The combined reported state appears in a new kind of object, one per (workload object, `Binding` object) pair.
 
 1. A user can request a list without aggregation, possibly after filtering, but certainly with a limit on list length. The expectation is that such a list makes sense only if the length of the list will be modest. For users that want access to the full reported state from each WEC for a large number of WECs, KubeStellar should have an abstraction that gives the users access --- in a functional way, not by making another copy --- to that state (which is already in the mailbox namespaces).
 
 1. The reported state for a given workload object from a given WEC is implicitly augmented with metadata about the WEC and about the end-to-end propagation from WDS to that WEC. This extra information is available just like the regular contents of the object, for use in combining reported state.
 
 ### Relation with SQL
 
-To a given workload object, the user can bind one or more named "status combiners". Each status combiner is analogous to an SQL SELECT statement that does the following things.
+To a given workload object, and in the context of a given `Binding` object, the user has bound some `StatusCollector` objects. The meaning of a `StatusCollector` in the context of a (workload object, `Binding` object) pair is analogous to an SQL SELECT statement that does the following things.
 
-1. The SELECT statement has one input table, and that has a row per WEC that the object is prescribed to propagate to.
+1. The SELECT statement has one input table, which has a row per WEC that the Binding says the workload object should go to.
 
 1. The SELECT statement can have a WHERE clause that filters out some of the rows.
 
 1. The SELECT statement either does aggregation or does not. In the case of not doing aggregation, the SELECT statement simply has a collection of named expressions defining the columns of its output.
 
 1. In the case of aggregation, the SELECT statement has the following.
 
-   - A `GROUP BY` clause saying how the rows (WECs) are grouped to
-     form the inputs for aggregation, in terms of named
-     expressions. For convenience here, each of these named
-     expressions is implicitly included in the output columns.
+    - An optional `GROUP BY` clause saying how the rows (WECs) are
+      grouped to form the inputs for aggregation, in terms of named
+      expressions. For convenience here, each of these named
+      expressions is implicitly included in the output columns.
 
-   - A collection of named expressions using aggregation functions to define
-     additional output columns.
+    - A collection of named expressions using aggregation functions to define
+      additional output columns.
 
 1. The SELECT statement has a LIMIT on the number of rows that it will yield.
 
+#### Relation with SQL, more precisely
```

Comment:
This is already sub section of relation with SQL. So maybe "Detailed description" ?

Reply:
Revised.